dipy/utils/parallel.py

```diff
@@ -42,17 +46,21 @@ def paramap(
     engine : str
         {"dask", "joblib", "ray", "serial"}
         The last one is useful for debugging -- runs the code without any
-        parallelization. Default: "joblib"
+        parallelization. Default: "ray"
     backend : str, optional
         What joblib or dask backend to use. For joblib, the default is "loky".
         For dask the default is "threading".
+    clean_spill : bool, optional
+        If True, clean up the spill directory after the computation is done.
+        Only applies to "ray" engine. Otherwise has no effect.
+        Default: True.
```

Comment:
Same as above concerning the default value.

Reply:
Done in 0140c8cf0